src/lib/genericEmbedPagination.ts

```diff
@@ -25,7 +25,7 @@ export async function createPaginatedEmbedHandler(initialInteraction: CommandInt
     }
 
     const filter = (buttonInteraction: Interaction) => initialInteraction.user.id === buttonInteraction.user.id;
-    const collector = message.createMessageComponentCollector({ filter, componentType: ComponentType.Button, time: 120_000 });
+    const collector = message.createMessageComponentCollector({ filter, componentType: ComponentType.Button, time: 5_000 });
```

Comment:
```suggestion
    const collector = message.createMessageComponentCollector({ filter, componentType: ComponentType.Button, time: 120_000 });
```

I assume this was changed for testing?